.github/workflows/gradle-test.yml

```
@@ -25,3 +25,6 @@ jobs:
 
     - name: Run Gradle test
       run: ./gradlew test
+
+    - name: Run Gradle assemble
+      run: ./gradlew assemble
```

This will catch javadocs issues

```suggestion
    - name: Run Gradle javadocs
      run: ./gradlew javadocs

    - name: Run Gradle assemble
      run: ./gradlew assemble